content/plugins/croustibat-jobs-monitor.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Jobs monitor
+slug: croustibat-jobs-monitor
+author_slug: croustibat
+categories: [developer-tools, analytics]
+description: Queue Monitoring Package for Filament
+docs_url: https://raw.githubusercontent.com/croustibat/filament-jobs-monitor/main/README.md
+github_repository: croustibat/filament-jobs-monitor
+has_dark_theme: true
+has_translations: true
+image: croustibat-jobs-monitor.jpg
+versions: [2]
+publish_date: 2023-08-09
```

Comment:
Please fix this so that it is the date of the first release of your plugin, rather than the date you submitted it here.